lite/examples/sound_classification/android_compose/app/src/main/java/org/tensorflow/lite/examples/soundclassifier/compose/ui/SoundClassifierScene.kt

```diff
@@ -0,0 +1,185 @@
+package org.tensorflow.lite.examples.soundclassifier.compose.ui
+
+import android.content.res.Configuration
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Devices
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.tensorflow.lite.examples.soundclassifier.compose.R
+import org.tensorflow.lite.examples.soundclassifier.compose.ui.theme.JetSoundClassifierTheme
+import org.tensorflow.lite.examples.soundclassifier.compose.ui.theme.gray800
+import org.tensorflow.lite.examples.soundclassifier.compose.ui.theme.orange500
+import org.tensorflow.lite.examples.soundclassifier.compose.ui.theme.progressColorPairs
+import org.tensorflow.lite.support.label.Category
+
+@Composable
+fun SoundClassifierScene(viewModel: SoundClassifierViewModel) {
```

Comment:
We usually call them `Screen` but not a strict convention

Reply:
Great point! Thanks.